.github/workflows/trivy-db-schedule.yml

```diff
@@ -0,0 +1,63 @@
+#
+# THIS FILE IS GENERATED, PLEASE DO NOT EDIT.
```

Comment:
Corresponding workflow template is forgotten.

Reply:
We've decided not to write any templates.
@shvgn